app/javascript/lib/MessageModal.js

```diff
@@ -15,6 +15,24 @@ export function showMessage(message, key, logProps) {
   functionHolder.showMessage(message, key, logProps)
 }
 
+/** take a json-api error object and render a modal with the contents.
+ * For now, this just uses the 'detail' field of each error.
+ */
+export function showJsonApiErrorMessage(error, key, logProps) {
```

Comment:
Out of curiosity,  does this trigger [an `error-modal` event log](https://mixpanel.com/s/3EDujS) as well?  How / why would this error modal logging differ from that error modal logging?